### PR TITLE
feat: make placeholder of cascader-view can be customize

### PR DESCRIPTION
**Pull request by liaoliao666**

### CODE REVIEW

**Comment by zombieJ** [comment]
Why not HOC of this?

**Comment by liaoliao666** [comment]
@zombieJ HOC will make more rerender than access the internal variable.

File: src/components/cascader-view/cascader-view.tsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ export const CascaderView: FC<CascaderViewProps> = p => {
               key={index.toString()}
               title={
                 <div className={`${classPrefix}-header-title`}>
-                  {selected ? selected.label : props.placeholder}
+                  {selected ? selected.label : typeof props.placeholder === "function" ? props.placeholder(index) : props.placeholder}
```
Comment (zombieJ) [inline src/components/cascader-view/cascader-view.tsx:131]: Could you help to add test case also?

**Comment by zombieJ** [comment]
> @zombieJ HOC will make more rerender than access the internal variable.

Could you provide a real case about this why it need `index` as args?

**Comment by miracles1919** [comment]
Please update the document

**Comment by liaoliao666** [comment]
@zombieJ 
image
Which placehoder are displayed base on region level

**Comment by liaoliao666** [comment]
@miracles1919 done

File: src/components/cascader-view/cascader-view.tsx (hunk 1 above)
Comment (liaoliao666) [reply src/components/cascader-view/cascader-view.tsx:131]: I can not install this project's dependencies in my company's internal network. So I can not run the test case. And I do not think u needs this test case for this simple case either.

**Comment by miracles1919** [comment]
I have modified the definition of ts based on your code

**Comment by zombieJ** [comment]
我更新一下 base 到 feature branch

**Review by zombieJ — APPROVED** [review]

**Pull request merged**